packages/server/graphql/types/AddMattermostAuthPayload.ts

```diff
@@ -0,0 +1,32 @@
+import {GraphQLObjectType} from 'graphql'
+import MattermostIntegration from './MattermostIntegration'
+import makeMutationPayload from './makeMutationPayload'
+import User from './User'
+import {GQLContext} from '../graphql'
+
+export const AddMattermostAuthSuccess = new GraphQLObjectType<any, GQLContext>({
+  name: 'AddMattermostAuthSuccess',
+  fields: () => ({
+    MattermostIntegration: {
+      type: MattermostIntegration,
```

Comment:
the success payload should try to have all responses be nonnull.
the error payload is the other payload type. that's why 
`AddMattermostAuthPayload = 'AddMattermostAuthSuccess | MutationError'`